src/shared/components/query/quickSearch/QuickSearch.tsx

```diff
@@ -232,7 +232,22 @@ export default class QuickSearch extends React.Component {
                     maxMenuHeight={550}
                     styles={{
                         dropdownIndicator: ()=>{ return { display:'none' } },
-                        control: (provided:any)=>{ return { ...provided, cursor:'text' } }
+                        control: (provided:any)=>{ return { ...provided, cursor:'text' } },
+                        option: (base:any, state:any) => {
+                            if (state.isSelected) {
+                                return {
+                                    ...base,
+                                    backgroundColor: 'inherit',
+                                    color: 'inherit',
+                                    ':hover': {
+                                        backgroundColor: '#DEEBFF'
```

Comment:
is there a reason we need to specify a hover color?  could we just use inherit?

Reply:
If we use inherit, the background color will be 'none' since its parent doesn't have the background color. So I set the color to which defined in the select2 library.